### PR TITLE
RHTAPBUGS-1146: TPA Dependency Bump

### PR DESCRIPTION
**Pull request by otaviof**
Using newer TPA Charts fixing GUAC related rollout issues.

### CODE REVIEW

**Comment by otaviof** [comment]
@flacatus, @xinredhat please have a look 🙏

**Comment by xinredhat** [comment]
I verified this bug and updated comments in [RHTAPBUGS-1146](https://issues.redhat.com//browse/RHTAPBUGS-1146). It fixed the cert issue.

File: bin/make.sh [hunk 1]
```diff
@@ -222,11 +222,6 @@ values() {
             PROMPT="0"
           fi
           ;;
-        TPA__*)
```
Comment (Roming22) [inline bin/make.sh:225]: @otaviof Why did you remove bit?

Comment (otaviof) [reply bin/make.sh:225]: With these lines in place all `TPA__*` was set to the single entry I was able to enter. I still need to investigate this further.
